ansible/roles/rhacm_assisted_installer/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+# Install assisted-installer onto a cluster with RHACM
+
+- name: Create directory for rhacm assisted-installer
+  file:
+    path: "{{ rhacm_install_directory }}/rhacm_assisted_installer"
+    state: directory
+
+- name: Copy files for rhacm assisted-installer
+  copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+  loop:
+  - src: assisted-installer.yml
+    dest: "{{ rhacm_install_directory }}/rhacm_assisted_installer/assisted-installer.yml"
+  - src: agentserviceconfig.yml
+    dest: "{{ rhacm_install_directory }}/rhacm_assisted_installer/agentserviceconfig.yml"
+
+- name: Template fix for assisted-service postgres permissions
+  template:
+    src: assisted-service-fix.sh
+    dest: "{{ rhacm_install_directory }}/rhacm_assisted_installer/assisted-service-fix.sh"
+    mode: "0744"
+
+- name: Patch provisioning configuration
+  shell: |
+    KUBECONFIG={{ hub_cluster_kubeconfig }} oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
+
+- name: Patch hive configuration
+  shell: |
+    KUBECONFIG={{ hub_cluster_kubeconfig }} oc patch hiveconfig hive  --type merge -p '{"spec":{"targetNamespace":"hive","logLevel":"debug","featureGates":{"custom":{"enabled":["AlphaAgentInstallStrategy"]},"featureSet":"Custom"} }}'
+
+- name: Create the assisted-installer and agentservconfig manifests
+  shell: |
+    KUBECONFIG={{ hub_cluster_kubeconfig }} oc apply -f {{ item }}
+  loop:
+  - "{{ rhacm_install_directory }}/rhacm_assisted_installer/assisted-installer.yml"
+  - "{{ rhacm_install_directory }}/rhacm_assisted_installer/agentserviceconfig.yml"
```

Comment:
`AgentServiceConfig` (`agentserviceconfigs.agent-install.openshift.io`) is a new CRD. You might want to wait for it to be available before you apply